scss/detail.scss

```diff
@@ -42,3 +47,8 @@
   height: 368px;
   width: 261px;
```

Comment:
Same

```
width: 100%;
max-width: 260px;
```

This will prevent this image from breaking the layout when the screen gets smaller than 260px;